alembic/versions/0595e877f471_add_streamphotometry_table.py

```diff
@@ -0,0 +1,63 @@
+"""Add StreamPhotometry table
+
+Revision ID: 0595e877f471
+Revises: c867ada3e0c4
+Create Date: 2021-04-23 10:34:12.985368
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '0595e877f471'
+down_revision = 'c867ada3e0c4'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table(
+        'stream_photometry',
+        sa.Column('created_at', sa.DateTime(), nullable=False),
+        sa.Column('modified', sa.DateTime(), nullable=False),
+        sa.Column('id', sa.Integer(), nullable=False),
+        sa.Column('stream_id', sa.Integer(), nullable=False),
+        sa.Column('photometr_id', sa.Integer(), nullable=False),
```

Comment:
Why this particular spelling here? Should this be `photometry_id`?

Reply:
It's `photometr_id`. Table names are pluralized in postgres (e.g. `objs`) and the last letter is left off in the join table to yield the singular, e.g. `obj_id`, and in this case we set the postgres name to `photometry` but the join table still has a letter left off (it assumes there's a trailing "s") -- this is the same for `GroupPhotometry`, which has a `photometr_id` column.